docker-compose.yaml

```
@@ -32,7 +32,9 @@ services:
     # depends_on:
     #   - postgres
     environment:
-      DATABASE_URL: "postgresql://emma:happy@172.17.0.2:5434/summafy?sslmode=disable"
+environment:
+      DATABASE_URL: ${DB_URL}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 6

(indentation)